src/components/helper.ts

```diff
@@ -0,0 +1,6 @@
+export function generateUniqueKey(
```

Comment:
this is dangerous as generate a new key on every call. if this is called within a Component, the Children would get rerendered every time the parent is. 
Are you sure?

Reply:
added this for a test, forgot to delete